src/Query.sql

```diff
@@ -24,10 +23,9 @@ Query- select CNAME from Customers where RATING = 100;
        | Pereira |
        +---------+
 
-
 3) Find the largest order taken by each Salesperson on each date.
 ------------------------------------------------------------------
-Query- select SNUM, ODATE, MAX(AMT) from Orders group by ODATE, SNUM;
+Query- SELECT SNUM, ODATE, MAX(AMT) FROM orders GROUP BY ODATE, SNUM;
```

Comment:
Please add name of sales person in this query.

Reply:
Done